packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -954,6 +954,7 @@ mixin WidgetInspectorService {
     _errorsSinceReload = 0;
   }
 
+  /// Whether structured errors are enabled.
```

Comment:
This would probably benefit from a little more detail. What does it mean to have "structured errors"?

Reply:
Right.  Will update.